src/development/tools/devtools/performance.md

```diff
@@ -153,8 +153,43 @@ enable the Track Layouts option:
 To see render object paint events in the timeline,
 enable the Track Paints option:
 
+
 ![Screenshot of track paints]({{site.url}}/assets/images/docs/tools/devtools/track-paints.png)
 
+## More debugging options
+
+To diagnose performance problems related to rendering layers,
+toggle off a rendering layer.
+These options are enabled by default.
+
+To see the effects on your app's performance,
+reproduce the activity in your app,
+select the new frames in the frames chart,
+to inspect the timeline events
+with the layers disabled.
+If Paint time or Raster time has significantly decreased,
```

Comment:
based on Dan's response above, it sounds like this should be just "If Raster time..." and not Paint